Error building on ubuntu

error[E0658]: macro dbg! is unstable (see issue #54306)                         
   --> /home/team4/.cargo/registry/src/github.com-1ecc6299db9ec823/narvie-cli-0.3.2/src/lib/instruction.rs:535:13
    |                                                                           
535 |             dbg!(memory_location);                                        
    |             ^^^^^^^^^^^^^^^^^^^^^^                                        
                                                                                
error: aborting due to previous error                                           
                                                                                
For more information about this error, try `rustc --explain E0658`.             
error: failed to compile `narvie-cli v0.3.2`, intermediate artifacts can be found at `/tmp/cargo-installNfHItu`

Caused by:
  Could not compile `narvie-cli`.

To learn more, run the command again with --verbose.

--- a/cli/src/lib/instruction.rs
+++ b/cli/src/lib/instruction.rs
@@ -532,7 +532,6 @@ where
                 ))?;
 
         if close_bracket_index != memory_location.len() - 1 {
-            dbg!(memory_location);
             return Err(InvalidArgument::MemoryLocation(
                 GetMemoryLocationError::TextAfterCloseParenthesis,
             ));

